distribution/src/conf/registry.xml

```diff
@@ -28,4 +28,24 @@
     <parameter name="GovRegRoot">{Root directory path for governance Registry}</parameter>
     <parameter name="LocalRegRoot">{Root directory path for local Registry}</parameter>
     -->
+
+    <!--
+        Since MicroIntegrator has a filed based registry, the following mapping is used to get the media-type of the file
+    -->
+    <parameter name="fileExtensionMappings"><![CDATA[
```

Comment:
It's not nice to have CDATA tag to specify xml elements. Shall we try to fix this?

Reply:
Moving file mappings to code